backend/app/mcp_server/tools/knowledge_external.py

```
@@ -171,6 +173,43 @@ def _search_rate_limit_status(user_id: int) -> ExternalMcpRateLimitStatus:
+def _load_namespace_map(db: Session, kbs: list) -> dict[str, Namespace]:
+    namespace_names = sorted(
+        {kb.namespace for kb in kbs if getattr(kb, "namespace", None) != "default"}
+    )
+    if not namespace_names:
+        return {}
+
+    namespaces = (
+        db.query(Namespace)
+        .filter(
+            Namespace.name.in_(namespace_names),
+            Namespace.is_active == True,
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)